lib/main_common.pm

```diff
@@ -501,12 +501,12 @@ sub load_extra_tests() {
             # sysauth test scenarios run in the console
             loadtest "sysauth/sssd";
         }
-        # finished console test and back to desktop
-        loadtest "console/consoletest_finish";
-        # kdump is not supported on aarch64, see BSC#990418
+        # kdump is not supported on aarch64, see bsc#1037407
         if (!check_var('ARCH', 'aarch64')) {
```

Comment:
Now that you have kdump_is_active function, you can also enable this for aarch64, we can at least start testing it.